app/assets/javascripts/src/Metamaps.Import.js

```diff
@@ -272,15 +263,22 @@ Metamaps.Import = {
       console.warn("Couldn't find metacode " + metacode_name + ' so used Wildcard instead.')
     }
 
+    var topic_permission = permission || Metamaps.Active.Map.get('permission')
+    var defer_to_map_id = permission === topic_permission ? Metamaps.Active.Map.get('id') : null
     var topic = new Metamaps.Backbone.Topic({
       name: name,
       metacode_id: metacode.id,
-      permission: permission || Metamaps.Active.Map.get('permission'),
-      desc: desc || "",
-      link: link
+      permission: topic_permission,
+      defer_to_map_id: defer_to_map_id,
+      desc: desc || ""
     })
+    topic.set('desc', desc || '') // TODO why is this necessary?
```

Comment:
these two overrides are required because Metamaps.Backbone.Topic has an init function which is currently overriding passed in values for Desc and Link. Going into Metamaps.Backbone.js and editing the init function for Topic will solve it